dev-python/influxdb/files/5.3.0-pandas-future-warning.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/influxdb/_dataframe_client.py b/influxdb/_dataframe_client.py
```

Comment:
Pop the link to the PR at the top of the file (I can do this on merge though).

Reply:
Done.